base/init-job.yaml

```diff
@@ -45,10 +45,13 @@ spec:
         volumeMounts:
         - name: client-certs
           mountPath: /cockroach-certs
+        env:
+        - name: COCKROACH_INIT_HOST
```

Comment:
Is this env var meant to be modified by users of these manifests? If so, should we have some docs about them?

Reply:
Yes it is. That's a good point. 
I'll add some info to the readme

Reply:
I've done this if you could take a look and let me know if this resolves the conversation. 
Thanks

Reply:
> I've done this if you could take a look and let me know if this resolves the conversation.
Thanks

It now looks like there's a lot more changes than just these environment variables, was that intentional?

Reply:
Yeah. I've just moved the original example to the `single-cluster` directory and added a `multi-cluster` directory. 
I've updated the links in the README to reflect the move

Reply:
> Yeah. I've just moved the original example to the `single-cluster` directory and added a `multi-cluster` directory. I've updated the links in the README to reflect the move

:+1: if you want to just get this change release it might be simpler to skip the examples for now and bring them in a separate PR (since they'll need review too), but up to you

Reply:
Nah. Might as well do a proper job

Reply:
> Nah. Might as well do a proper job

I think the note in the README is a reasonable job for this :slightly_smiling_face:

:thinking:  the new examples might need some extra discussion around what we want/need here. Chatting with @sbuliarca we realised we didn't know there were already examples in this repo but now we're questioning the value of them, since the examples aren't actually running anywhere they can:

* Quickly get out of date, since nothing checks they produce a valid application (even if originally they were based on something running in production)
* Might poorly reflect real-world use cases (since they don't actually see real-world usage)

So I'm leaning towards just leaving these out for now until we're sure what we want from examples in this repo. What do you think?

Thinking out-load: maybe a nicer alternative would be to have 'canonical' examples from what is actually running in production that we can point to instead (but this is a conversation to be had separately).

Reply:
Ok. Sounds reasonable. I've removed the examples so we're back to three files changed including the README.